.github/workflows/tag-release.yml

```diff
@@ -0,0 +1,44 @@
+name: Tag & GitHub release
+
+on:
+  pull_request_target:
+    branches:
+      - main
+    types: [closed]
+
+jobs:
+  tag-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        if: github.event.pull_request.merged == true
+        with:
+          fetch-depth: 0
+
+      - name: Extract version
+        if: github.event.pull_request.merged == true
+        id: version
+        run: |
+          NEW_VERSION=$(node -p 'require("./package.json").version')
+          echo "::set-output name=version::v$NEW_VERSION"
+          echo "${{ github.event.pull_request.body }}" > CHANGELOG.md
+
+      - name: Create a git tag
+        if: github.event.pull_request.merged == true
+        run: git tag $NEW_VERSION && git push --tags
+        env:
+          NEW_VERSION: ${{ steps.version.outputs.version }}
```

Comment:
Does `$NEW_VERSION` contain the prefix `v` here even if we set the environment variable after?

Reply:
Ah, right, good catch. Let me add a `v`.

Reply:
Checked, it works correctly like it is.

Reply:
Testing this in a private repo it seems to include the prefix